### ISSUE TITLE
Add React 18 as peer dependency.

### ISSUE THREAD

**Issue by syhily** [immerjs/use-immer#98]
```log
npm ERR! Found: react@18.0.0
npm ERR! node_modules/react
npm ERR!   react@"^18.0.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.8.0 || ^17.0.1" from use-immer@0.6.0
npm ERR! node_modules/use-immer
npm ERR!   use-immer@"^0.6.0" from the root project
```

**Comment by mweststrate** [comment]
PR welcome

**Comment by w3bdesign** [comment]
@syhily 

https://github.com/immerjs/use-immer/pull/99

**Comment by w3bdesign** [comment]
@mweststrate @syhily 

https://github.com/immerjs/use-immer/pull/99

**Comment by wereHamster** [comment]
Can we get another release that includes the PR/commit?

**Comment by wereHamster** [comment]
It seems as if tag [0.7.0](https://www.npmjs.com/package/use-immer/v/0.7.0) includes the change but the tag itself not available on github. Did you forget to push it?

**Comment by baringji** [comment]
Agree, tag/release not available on github

**Comment by mweststrate** [comment]
Was released in 0.7
